tests/test_meshingmode/test_meshing_launch.py

```diff
@@ -57,7 +57,7 @@ def test_launch_pure_meshing(load_mixing_elbow_pure_meshing):
     with open(file_path) as fp:
         for count, line in enumerate(fp):
             pass
-    assert count == 17
+    assert count == 18
```

Comment:
@akulshre-qa 
As a general principle, I would prefer to avoid asserting such a thing, which is only weakly correlated with the correct functioning of the journal recording.  The assertion could succeed while journaling has failed and vice versa. In more concrete terms, we are currently changing the implementation, which will increase the number of lines in the file.

Reply:
Hi @seanpearsonuk thanks for the feedback. I just removed this assertion. We are anyway validating this recorded journal by replaying it back in the same session.